_episodes/16-parallel.md

```diff
@@ -40,45 +405,23 @@ in the process. In the context of our queuing system, however, we do not need to
 this information, the `mpirun` command will obtain it from the queuing system, by
```

Comment:
`mpirun` needs to be configurable. The standard is actually `mpiexec` but even then there are systems (like ours) that *require* other MPI launchers (`srun`, `aprun`,...)...`{{ site.remote.mpiexec }}`?

Reply:
I've replaced all the `mpirun` commands with `mpiexec` -- I'm good to make it configurable for `srun` or other commands, whether in a snippet or a variable. Could that be a future issue and PR if the remainder of the episode is acceptable?